Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/BayesQuasi.py

```diff
@@ -3,13 +3,12 @@
 import numpy as np
 
 from IndirectImport import *
-
 from mantid.api import (PythonAlgorithm, AlgorithmFactory, MatrixWorkspaceProperty, PropertyMode,
                         WorkspaceGroupProperty, Progress)
 from mantid.kernel import StringListValidator, Direction
 from mantid.simpleapi import *
 from mantid import config, logger
-
+from IndirectCommon import *
```

Comment:
Check how many `IndirectCommon` imports are required, if not too many then may be worth explicitly saying which we want rather than importing all (*)